gm4/plugins/output.py

```diff
@@ -68,7 +68,26 @@ def release(ctx: Context):
 
 	# publish to download platforms
 	publish_modrinth(ctx, release_dir, file_name)
-	publish_smithed(ctx, release_dir, file_name)
+	publish_smithed(ctx, file_name)
+
+
+def libraries(ctx: Context):
+	"""Saves the zipped standalong library to FIXME"""
```

Comment:
I would assume `FIXME` is a placeholder?

Reply:
Yes that's correct. I was planning to save the build to `release/libraries/lib_machines` but I'm not so sure about that.